includes/html/pages/device/ports.inc.php

```diff
@@ -101,18 +101,18 @@
     echo "<div style='display: block; clear: both; margin: auto; min-height: 500px;'>";
     unset($seperator);
 
-    // FIXME - FIX THIS. UGLY.
+    // FIXME - FIX THIS. UGLY. 
     foreach (dbFetchRows('select * from ports WHERE device_id = ? AND `disabled` = 0 ORDER BY ifIndex', array($device['device_id'])) as $port) {
         $port = cleanPort($port, $device);
-        echo "<div style='display: block; padding: 3px; margin: 3px; min-width: 183px; max-width:183px; min-height:90px; max-height:90px; text-align: center; float: left; background-color: #e9e9e9;'>
-            <div style='font-weight: bold;'>".makeshortif($port['ifDescr']).'</div>
-            <a href="'.generate_port_url($port)."\" onmouseover=\"return overlib('\
-            <div style=\'background-color: #ffffff;\'>\
-            <div style=\'font-size: 16px; padding:5px; font-weight: bold; color: #555;\'>".$device['hostname'].' - '.$port['ifDescr'].'</div>\
-            '.$port['ifAlias']." \
-            <img src=\'graph.php?type=".$graph_type.'&amp;id='.$port['port_id'].'&amp;from='.$from.'&amp;to='.$config['time']['now']."&amp;width=450&amp;height=150\'>\
+        echo "<div class='minigraph-div'><div style='font-weight: bold;'>".makeshortif($port['ifDescr'])."</div>
+            <a href=\"'.generate_port_url($port).'\" onmouseover=\"return overlib('<div class=\'overlib-content\'>\
+      	    <div class=\'overlib-text\'>".$device['hostname']." - ".$port['ifDescr']."</div>\
+            <span class=\'overlib-title\'>".$port['ifAlias']."</span>\
+            <img src=\'graph.php?type=".$graph_type.'&amp;id='.$port['port_id'].'&amp;from='.$from.'&amp;to='.$config['time']['now']."&amp;width=450&amp;heig\
+ht=150\'>\
             </div>\
-            ', CENTER, LEFT, FGCOLOR, '#e5e5e5', BGCOLOR, '#e5e5e5', WIDTH, 400, HEIGHT, 150);\" onmouseout=\"return nd();\"  >"."<img src='graph.php?type=".$graph_type.'&amp;id='.$port['port_id'].'&amp;from='.$from.'&amp;to='.$config['time']['now']."&amp;width=180&amp;height=45&amp;legend=no'>
+            ', CENTER, LEFT, FGCOLOR, '#e5e5e5', BGCOLOR, '#e5e5e5', WIDTH, 400, HEIGHT, 150);\" onmouseout=\"return nd();\"  >"."<img src='graph.php?type=".\
```

Comment:
There is an extra `\` at the end of this line.

Reply:
Ok changed  (copy & paste issue to git webui ! my bad)